verification/verification.py

```diff
@@ -0,0 +1,68 @@
+TOKEN = 'a5af923504bc221d004134f82eff8d5ad6f74a32'
```

Comment:
Your pull request: REQUESTS-1021 Added verification.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']

Your commit: VERIFICATION-1021 Added verification.py
The prefix of your comment must be from ['LEETCODE', 'GENERATOR', 'TRIANGLE', 'HEXNUMBER']